tests/ErrorTest.php

```diff
@@ -4,18 +4,23 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 use AWS\CRT\CRT;
+use Yoast\PHPUnitPolyfills\TestCases\TestCase;
 
-final class ErrorTest extends PHPUnit_Framework_TestCase {
+final class ErrorTest extends TestCase
+{
 
-    public function testNoInitialError() {
+    public function testNoInitialError()
+    {
```

Comment:
Consistent braces

Reply:
aws-sdk-php using the other type of brace style. Maybe we'd better to follow them. 

But, it's just too much here to change. Let's keep it as you suggested, and we can update our format check for php in the future to adjust them.